odoo/addons/base/models/res_users.py

```diff
@@ -1522,7 +1522,7 @@ def change_password(self):
 
 class ResUsersApikeys(models.Model):
     _name = 'res.users.apikeys'
-    _description = 'Users API Keys'
+    _description = 'Users API Key'
```

Comment:
Shouldn't it be singular too?
```suggestion
    _description = 'User API Key'
```